content/add-ons/rivet-datepicker.md

```diff
@@ -0,0 +1,235 @@
+
+The Rivet datepicker add-on uses the `aria-describedby` attribute to associate format instructions with datepicker fields.
+
+{{< code lang="html" >}}<fieldset aria-describedby="hs-date-hint">
+  <legend class="rvt-ts-18 rvt-text-bold">When did you attend highschool?</legend>
+  <span class="rvt-ts-14" id="hs-date-hint">Type dates in the format: mm-dd-yyyy</span>
+  <div class="rvt-grid rvt-m-top-xl">
+    <div class="rvt-grid__item-4-md-up">
+      <div class="rvt-datepicker rvt-m-bottom-sm">
+        <label for="start">Start date</label>
+        <div class="rvt-datepicker__input" aria-hidden="true">
+          <input autocomplete="off" type="text" id="start" aria-describedby="hs-date-hint">
+          <div class="rvt-datepicker__icon">
+            <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
+              <path fill="currentColor" d="M12.29,2H12V1a1,1,0,0,0-2,0V2H6V1A1,1,0,0,0,4,1V2H3.71A2.78,2.78,0,0,0,1,4.83v7.33A2.78,2.78,0,0,0,3.71,15h8.57A2.78,2.78,0,0,0,15,12.17V4.83A2.78,2.78,0,0,0,12.29,2ZM3.71,4H4V5H6V4h4V5h2V4h.29a.78.78,0,0,1,.71.83V7H3V4.83A.78.78,0,0,1,3.71,4Zm8.57,9H3.71A.78.78,0,0,1,3,12.17V9H13v3.17A.78.78,0,0,1,12.29,13Z"/>
+            </svg>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="rvt-grid__item-4-md-up">
+      <div class="rvt-datepicker">
+        <label for="end">End date</label>
+        <div class="rvt-datepicker__input" aria-hidden="true">
+          <input autocomplete="off" type="text" id="end" aria-describedby="hs-date-hint">
+          <div class="rvt-datepicker__icon">
+            <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
+              <path fill="currentColor" d="M12.29,2H12V1a1,1,0,0,0-2,0V2H6V1A1,1,0,0,0,4,1V2H3.71A2.78,2.78,0,0,0,1,4.83v7.33A2.78,2.78,0,0,0,3.71,15h8.57A2.78,2.78,0,0,0,15,12.17V4.83A2.78,2.78,0,0,0,12.29,2ZM3.71,4H4V5H6V4h4V5h2V4h.29a.78.78,0,0,1,.71.83V7H3V4.83A.78.78,0,0,1,3.71,4Zm8.57,9H3.71A.78.78,0,0,1,3,12.17V9H13v3.17A.78.78,0,0,1,12.29,13Z"/>
+            </svg>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</fieldset>
+{{< /code >}}
+
+{{< alert variant="warning" title="Allowing manual date entry" >}}
+It's important to note that if you are using a calendar date picker, make sure that you also provide instructions that let users know the format in which they should type the date manually in to the input.
+{{< /alert >}}
+
+### Initializing a calendar datepicker
+1. Include rivet-datepicker.js in your page via a `<script>` tag or your build process.
+2. Include the rivet-datepicker.css file in your app.
+3. Create a new instance of RivetDatepicker for each calendar datepicker input field that appears on your page.
+
+For example, if your page contains the following input:
+
+{{< code lang="html" >}}
```

Comment:
These opening shortcode tags (from here down) need to be on the same line as the first line of code in the snippet so that hugo doesn't render an extra line of whitespace at the top.